app/assets/javascripts/directives/buddy_request_box.js

```diff
@@ -0,0 +1,101 @@
+'use strict';
```

Comment:
This won't do anything, right? `"use strict";` would need to be right at the top of `application.js` and would then affect the entire application.